src/converse-chatview.js

```diff
@@ -70,7 +70,8 @@ converse.plugins.add('converse-chatview', {
                 'call': false,
                 'clear': true,
                 'emoji': true,
-                'spoiler': true
+                'spoiler': true,
+                'download': true
```

Comment:
This can go into src/converse-attachement-download.js instead, as well as the other changes here, so that `converse-chatview` doesn't have to be modified at all.

That way you can easily make this feature a separate plugin in the `community-modules` repo.